suggestedimages/wikidata/queries/__init__.py

```diff
@@ -1,20 +1,43 @@
+from typing import *
+
 from .util import bind_sparql_query, Identifier
 
-def label_or_alias_capitalized_or_not(text, language, limit=50):
+
+def label_or_alias_capitalized_or_not(text: str, language: str, limit=50):
     return bind_sparql_query('''
-SELECT distinct ?item ?itemLabel ?itemDescription WHERE{
+SELECT DISTINCT ?item WHERE{
   VALUES ?prefLabel {
     {{text}}@{{language}}
     {{text_capitalized}}@{{language}}
   }
 
-  ?item rdfs:label|skos:altLabel ?prefLabel
+  ?item rdfs:label|skos:altLabel ?prefLabel .
 }
 
 LIMIT {{limit}}
 ''',
         text = text,
         text_capitalized = text.capitalize(),
         language = Identifier(language),
-        limit = 50
+        limit = limit
+    )
+
+
+
+def property_has_any_of_values(property: str, values: list[str], limit=50):
+    """Return items where property `property` has any of values `values`.
+    """
```

Comment:
This comment might be redundant. The function name is already as (if not more) clear on what the purpose of it is.

Reply:
:+1: